tests/integration_tests/reports/commands_tests.py

```diff
@@ -767,11 +763,9 @@ def test_email_chart_report_schedule_force_screenshot(
         )
         # assert that the link sent is correct
         assert (
-            '<a href="http://0.0.0.0:8080/explore/?'
-            "form_data=%7B%22slice_id%22%3A%20"
-            f"{create_report_email_chart_force_screenshot.chart.id}%7D&"
-            'standalone=0&force=true">Explore in Superset</a>'
-            in email_mock.call_args[0][2]
+            '<a href="http://0.0.0.0:8080/explore/?form_data=%7B%22slice_id%22%3A+'
+            f"{create_report_email_chart_force_screenshot.chart.id}"
+            '%7D&force=true">Explore in Superset</a>' in email_mock.call_args[0][2]
```

Comment:
aside from removing standalone=0, which doesn't do anything in the page load, the changes also turned `slice_id%22%3A%20` into `slice_id%22%3A+` but when I tested this url live from a report, it still renders the chart correctly.